### DISCUSSION TITLE
dev server works with web component but built does not

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#11249] [CLI - PWA mode]
In testing on my local dev server (`quasar dev`) an imported web component in my layout works fine, but when I issue the build command (`quasar build -m pwa`) and upload to my server it does not and I can't even find reference to it in the build's source files (while I DO see reference to in in local dev server while using web inspection tools). There is no error produced while building. Is there a way to figure out what is different in the way the local dev server builds vs the quasar build command?

**Comment by metalsadman** [comment]
you can try building with debug flag on, ie. `quasar build -m pwa --debug`, see if that logs something.
